app/views/spree/admin/orders/_form/_adjustments.html.haml

```diff
@@ -0,0 +1,14 @@
+- if adjustments.present?
```

Comment:
what if instead, we move this conditional to wrap the lines 7 and 8 of app/views/spree/admin/orders/_form.html.haml? this would decouple this view from the details of adjustments and make it more maintainable. It feels like a responsibility that _adjustments.html.haml shouldn't be aware of and can save us some CPU cycles.

Reply:
we would require 2 if statements in _form (see that "adjustments" here are different lists in _form)
if in spree v2.2. with shipment_adjustments, that would make 3 if statements.

Reply:
Yes, I know but that is something we would need to fix in `_form`. If it gets a bit messy is not `_adjustments.html.haml` responsibility.

Reply:
I created https://github.com/openfoodfoundation/openfoodnetwork/issues/3475 for this.

Reply:
:+1: